cmd/utils/flags.go

```diff
@@ -876,6 +876,11 @@ var (
 		Usage:    "Allow for unprotected (non EIP155 signed) transactions to be submitted via RPC",
 		Category: flags.APICategory,
 	}
+	EnablePrivateTxRPC = &cli.BoolFlag{
+		Name:     "rpc.enable-private-tx",
```

Comment:
The `EnablePrivateTxRPC` seems not very intuitive. How about `PrivateTxMode` and `rpc.private-tx-mode`?